### PR TITLE
update(Divider): Add short prop

### PR DESCRIPTION
**Pull request by schillerk**
to: @milesj @stefhatcher @hayes @alecklandgraf

## Description
A lot of @kenchendesign's designs involve a short divider, it's a pretty small component and I'm happy to just write it for individual apps, but seems like it could be a good fit for this Component.
https://pages.git.musta.ch/ken-chen/airbnb-prototypes/prototype-kit/

I could alternatively make this a `width` prop that takes in an arbitrary value, but in the interest of being opinionated and consistent I think a single option is more appropriate.].

## Screenshots
It's just this line haha.
Screen Shot 2019-07-22 at 2 24 11 PM

## Checklist

- My code follows the style guide of this project.
- I have updated or added documentation accordingly.
- I have read the CONTRIBUTING document.

### CODE REVIEW

File: packages/core/src/components/Divider/index.tsx [hunk 1]
```diff
@@ -7,28 +7,34 @@ export type Props = {
...
   };
 
   render() {
-    const { cx, styles, bottom, top } = this.props;
+    const { cx, styles, bottom, short, top } = this.props;
 
     return (
       <Spacing bottom={bottom} top={top}>
-        <div className={cx(styles.divider)} />
+        <div className={cx(styles.divider, short && styles.divider_short)} />
```
Comment (stefhatcher) [inline packages/core/src/components/Divider/index.tsx]: can you add a test to test that the className has changed when short is passed in ? an example would be, with `short` passed on the wrapper component:

```
expect(wrapper.prop('className')).toMatch('divider_short');
```

**Comment by stefhatcher** [comment]
> It's just this line haha.

🤣 love it

File: packages/core/test/components/Divider.test.tsx [hunk 2]
```diff
@@ -15,4 +15,10 @@ describe('<Divider />', () => {
     expect(wrapper.prop('bottom')).toBe(4);
     expect(wrapper.prop('top')).toBe(2);
   });
+
+  it('can pass top and bottom to spacing', () => {
```
Comment (stefhatcher) [inline packages/core/test/components/Divider.test.tsx]: hrmm maybe a different test message here? `it('renders a short divider'`?

Comment (schillerk) [reply packages/core/test/components/Divider.test.tsx]: Ack, meant to change this, thanks

**Pull request merged**